docs/.patch/sphinx-versioning.patch

```diff
@@ -15,7 +15,7 @@ index a90587d4..1d58117a 100644
 @@ -6,7 +6,12 @@
  import os
  import sys
- 
+
```

Comment:
this hunk must be removed, the empty space is part of the patched file!

Reply:
ups... my script to clean EOL and EOF did this

Reply:
the line is still there, what the script removed was a trailing white space, not the line it self , check the file

Reply:
yes, the trailing white space has been removed, that's the issue, 
but patch spec must match exactly the content of the time to be applied successfully, so it shouldn't be modified

Reply:
ok , restored